src/components/common/Select.js

```diff
@@ -10,6 +10,7 @@ export default class Select extends PureComponent {
     type: string.isRequired,
     placeholder: string,
     meta: object,
+    options: Array
```

Comment:
`Array` isn't a proptype 🤔

Reply:
Please change this

Reply:
@mdamonte88 This one is still not fixed, you have array, and arrayOf(), but not Array. And you can see there also that you are not importing Array from any dependency, you are using Javascript's Array object.